content/_components/shopify-admin-v2/index.md

```diff
@@ -425,55 +427,45 @@ query{
 }
 ```
 
-1. Select in `Object type` option `Customers`
-
+**1.** Select in `Object type` option `Customers`.
 {% include img.html max-width="100%" url="img/example-1.png" title="Example 1" %}
 
-2. From dropdown `Select basic fields for resulting object` select basic fields: `id`, `state` and `firstName`.
-
+**2.** From dropdown `Select basic fields for resulting object` select basic fields: `id`, `state` and `firstName`.
 {% include img.html max-width="100%" url="img/example-2.png" title="Example 2" %}
-
 >**Please Note**: Basic fields are fields that can be requested without specifying arguments or adding additional levels: `id`, `email`, etc.
 
-3. Paste additional fields in field `You can provide additional fields here`:
-
+**3.** Paste additional fields in field `You can provide additional fields here`:
 {% include img.html max-width="100%" url="img/example-3.png" title="Example 3" %}
-
 > **Please Note**: Additional fields are fields that can't be selected in `Basic fields` section. As rule, they are fields with specifying arguments or adding additional levels such as: `addresses{country}`.
 
-4. Select `Emit Behavior`. If it is needed to receive each object individually, use `Emit individually`, in other case, `Emit page` option - all objects will be returned in one array `result`. For example, lets use 'Emit individually'
-
+**4.** Select `Emit Behavior`. If it is needed to receive each object individually, use `Emit individually`, in other case, `Emit page` option - all objects will be returned in one array `result`. For example, lets use 'Emit individually'.
```

Comment:
**4.** Select `Emit Behavior`. If it is needed to receive each object individually, use `Emit individually`, in other cases, `Emit page` option - all objects will be returned in one array `result`. For example, let's use 'Emit individually'.